src/html/options.html

```diff
@@ -57,6 +57,15 @@
 	<span>and Drag for <input type="text" size="2" id="Activation_Min"> pixels</span>
 </label>
 
+<label>
+	<span>Default Action</span>
+	<select id="DefaultAction" title="Select the default action.">
+		<option value="1">Open Links</option>
```

Comment:
I'm not sure how it could be done but ideally, these would reference the same constants.

Reply:
the only solution I can think of is to store in DefaultAction preference the strings "OPEN_LINKS" and "COPY_LINKS" instead of 1 and 2

Reply:
I'm not sure how this could be done either without it being a pain, lets let it be for now.